osbenchmark/results_publisher.py

```diff
@@ -145,16 +162,33 @@ def publish(self):
 
         metrics_table.extend(self._publish_transform_stats(stats))
 
+        max_throughput = -1
```

Comment:
Nit: Would also add a comment above this to specify that the following three variables are related to `clients_list` parameter in test_procedures. Newcomers might not quickly understand that this is only relevant to throughput testing clients